2013/07/26 19:15:46 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "gii/page".' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/page')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\xampp\htdocs\portfolio\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/portfolio/index.php?r=gii/page&view=about
---
2013/07/26 19:26:44 [error] [exception.CException] exception 'CException' with message 'CAssetManager.basePath "C:\xampp\htdocs\portfolio\assets" is invalid. Please make sure the directory exists and is writable by the Web server process.' in C:\xampp\htdocs\yii\framework\web\CAssetManager.php:138
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CAssetManager.php(123): CAssetManager->setBasePath('C:\xampp\htdocs...')
#1 C:\xampp\htdocs\yii\framework\web\CAssetManager.php(217): CAssetManager->getBasePath()
#2 C:\xampp\htdocs\yii\framework\gii\GiiModule.php(142): CAssetManager->publish('C:\xampp\htdocs...')
#3 C:\xampp\htdocs\yii\framework\base\CComponent.php(111): GiiModule->getAssetsUrl()
#4 C:\xampp\htdocs\yii\framework\base\CModule.php(105): CComponent->__get('assetsUrl')
#5 C:\xampp\htdocs\yii\framework\gii\views\layouts\main.php(5): CModule->__get('assetsUrl')
#6 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#7 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#8 C:\xampp\htdocs\yii\framework\web\widgets\CContentDecorator.php(76): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#9 C:\xampp\htdocs\yii\framework\web\widgets\CContentDecorator.php(54): CContentDecorator->decorate('<div class="con...')
#10 C:\xampp\htdocs\yii\framework\web\widgets\COutputProcessor.php(44): CContentDecorator->processOutput('<div class="con...')
#11 C:\xampp\htdocs\yii\framework\web\CBaseController.php(206): COutputProcessor->run()
#12 C:\xampp\htdocs\yii\framework\web\CBaseController.php(300): CBaseController->endWidget('CContentDecorat...')
#13 C:\xampp\htdocs\yii\framework\gii\views\layouts\column1.php(7): CBaseController->endContent()
#14 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#15 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#16 C:\xampp\htdocs\yii\framework\web\CController.php(784): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#17 C:\xampp\htdocs\yii\framework\gii\controllers\DefaultController.php(47): CController->render('login', Array)
#18 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DefaultController->actionLogin()
#19 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#20 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#21 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#22 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('login')
#23 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/default/log...')
#24 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#25 C:\xampp\htdocs\portfolio\index.php(13): CApplication->run()
#26 {main}
REQUEST_URI=/portfolio/index.php?r=gii/default/login
---
2013/07/26 19:33:16 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [1049] Unknown database 'portfolio'' in C:\xampp\htdocs\yii\framework\db\CDbConnection.php:381
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 C:\xampp\htdocs\yii\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\yii\framework\base\CModule.php(387): CDbConnection->init()
#3 C:\xampp\htdocs\yii\framework\base\CModule.php(103): CModule->getComponent('db')
#4 C:\xampp\htdocs\yii\framework\gii\generators\model\ModelCode.php(57): CModule->__get('db')
#5 C:\xampp\htdocs\yii\framework\web\CFormModel.php(39): ModelCode->init()
#6 C:\xampp\htdocs\yii\framework\gii\CCodeGenerator.php(153): CFormModel->__construct()
#7 C:\xampp\htdocs\yii\framework\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#15 C:\xampp\htdocs\portfolio\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/portfolio/index.php?r=gii/model
HTTP_REFERER=http://localhost/portfolio/index.php?r=gii/default/index
---
2013/07/26 19:34:09 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [1049] Unknown database 'portfolio'' in C:\xampp\htdocs\yii\framework\db\CDbConnection.php:381
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 C:\xampp\htdocs\yii\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\yii\framework\base\CModule.php(387): CDbConnection->init()
#3 C:\xampp\htdocs\yii\framework\base\CModule.php(103): CModule->getComponent('db')
#4 C:\xampp\htdocs\yii\framework\gii\generators\model\ModelCode.php(57): CModule->__get('db')
#5 C:\xampp\htdocs\yii\framework\web\CFormModel.php(39): ModelCode->init()
#6 C:\xampp\htdocs\yii\framework\gii\CCodeGenerator.php(153): CFormModel->__construct()
#7 C:\xampp\htdocs\yii\framework\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#15 C:\xampp\htdocs\portfolio\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/portfolio/index.php?r=gii/model
---
2013/07/26 19:58:41 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:179
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('imagenesBlog/ad...')
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\xampp\htdocs\portfolio\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/portfolio/index.php?r=imagenesBlog/admin
HTTP_REFERER=http://localhost/portfolio/index.php?r=imagenesBlog/create
---
2013/07/28 19:35:27 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "gii/index".' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/index')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\xampp\htdocs\portfolio\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/portfolio/index.php?r=gii/index
---
2013/07/28 19:35:32 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "imagenesBlog/index".' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('imagenesBlog/in...')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\xampp\htdocs\portfolio\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/portfolio/index.php?r=imagenesBlog/index
---
2013/07/28 19:35:39 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "gii/index".' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/index')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\xampp\htdocs\portfolio\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/portfolio/index.php?r=gii/index
---
2013/07/28 19:51:25 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "login".' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('login')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('login')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('blog/login')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\portfolio\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/portfolio/index.php?r=blog/login
---
2013/07/28 19:51:36 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "edir".' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('edir')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('edir')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('blog/edir')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\portfolio\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/portfolio/index.php?r=blog/edir
---
2013/07/28 19:51:45 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "edit.php".' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('edit.php')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('edit.php')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('blog/edit.php')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\portfolio\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/portfolio/index.php?r=blog/edit.php
---
2013/07/28 22:23:08 [error] [exception.CException] exception 'CException' with message 'Property "CActiveForm.enctype" is not defined.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:173
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWidgetFactory.php(161): CComponent->__set('enctype', 'multipart/form-...')
#1 C:\xampp\htdocs\yii\framework\web\CBaseController.php(146): CWidgetFactory->createWidget(Object(BlogController), 'CActiveForm', Array)
#2 C:\xampp\htdocs\yii\framework\web\CBaseController.php(189): CBaseController->createWidget('CActiveForm', Array)
#3 C:\xampp\htdocs\portfolio\protected\views\blog\_form.php(13): CBaseController->beginWidget('CActiveForm', Array)
#4 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#5 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#6 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#7 C:\xampp\htdocs\portfolio\protected\views\blog\create.php(18): CController->renderPartial('_form', Array)
#8 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#9 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#10 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#11 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('create', Array, true)
#12 C:\xampp\htdocs\portfolio\protected\controllers\BlogController.php(79): CController->render('create', Array)
#13 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): BlogController->actionCreate()
#14 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#15 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#16 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#17 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#18 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#19 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#20 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#21 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#22 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('create')
#23 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('blog/create')
#24 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#25 C:\xampp\htdocs\portfolio\index.php(13): CApplication->run()
#26 {main}
REQUEST_URI=/portfolio/index.php?r=blog/create
---
2013/07/28 22:50:05 [error] [php] Undefined variable: cs (C:\xampp\htdocs\portfolio\protected\views\blog\_form.php:5)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(869): BlogController->renderFile()
#1 C:\xampp\htdocs\portfolio\protected\views\blog\create.php(18): BlogController->renderPartial()
#2 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require()
#3 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): BlogController->renderInternal()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(869): BlogController->renderFile()
#5 C:\xampp\htdocs\yii\framework\web\CController.php(782): BlogController->renderPartial()
#6 C:\xampp\htdocs\portfolio\protected\controllers\BlogController.php(79): BlogController->render()
#7 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): BlogController->actionCreate()
#8 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#9 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): BlogController->runAction()
#10 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#12 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): BlogController->filterAccessControl()
#13 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#14 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#15 C:\xampp\htdocs\yii\framework\web\CController.php(265): BlogController->runActionWithFilters()
#16 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): BlogController->run()
#17 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#18 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#19 C:\xampp\htdocs\portfolio\index.php(13): CWebApplication->run()
REQUEST_URI=/portfolio/index.php?r=blog/create
in C:\xampp\htdocs\portfolio\protected\views\blog\_form.php (5)
in C:\xampp\htdocs\portfolio\protected\views\blog\create.php (18)
in C:\xampp\htdocs\portfolio\protected\controllers\BlogController.php (79)
2013/07/28 22:53:01 [error] [php] Trying to get property of non-object (C:\xampp\htdocs\portfolio\protected\views\blog\_form.php:6)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(869): BlogController->renderFile()
#1 C:\xampp\htdocs\portfolio\protected\views\blog\create.php(18): BlogController->renderPartial()
#2 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require()
#3 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): BlogController->renderInternal()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(869): BlogController->renderFile()
#5 C:\xampp\htdocs\yii\framework\web\CController.php(782): BlogController->renderPartial()
#6 C:\xampp\htdocs\portfolio\protected\controllers\BlogController.php(79): BlogController->render()
#7 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): BlogController->actionCreate()
#8 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#9 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): BlogController->runAction()
#10 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#12 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): BlogController->filterAccessControl()
#13 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#14 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#15 C:\xampp\htdocs\yii\framework\web\CController.php(265): BlogController->runActionWithFilters()
#16 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): BlogController->run()
#17 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#18 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#19 C:\xampp\htdocs\portfolio\index.php(13): CWebApplication->run()
REQUEST_URI=/portfolio/index.php?r=blog/create
in C:\xampp\htdocs\portfolio\protected\views\blog\_form.php (6)
in C:\xampp\htdocs\portfolio\protected\views\blog\create.php (18)
in C:\xampp\htdocs\portfolio\protected\controllers\BlogController.php (79)
2013/07/28 23:02:35 [error] [exception.CException] exception 'CException' with message 'Property "CActiveForm.enctype" is not defined.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:173
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWidgetFactory.php(161): CComponent->__set('enctype', 'multipart/form-...')
#1 C:\xampp\htdocs\yii\framework\web\CBaseController.php(146): CWidgetFactory->createWidget(Object(BlogController), 'CActiveForm', Array)
#2 C:\xampp\htdocs\yii\framework\web\CBaseController.php(189): CBaseController->createWidget('CActiveForm', Array)
#3 C:\xampp\htdocs\portfolio\protected\views\blog\_form.php(15): CBaseController->beginWidget('CActiveForm', Array)
#4 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#5 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#6 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#7 C:\xampp\htdocs\portfolio\protected\views\blog\create.php(18): CController->renderPartial('_form', Array)
#8 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#9 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#10 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#11 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('create', Array, true)
#12 C:\xampp\htdocs\portfolio\protected\controllers\BlogController.php(79): CController->render('create', Array)
#13 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): BlogController->actionCreate()
#14 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#15 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#16 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#17 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#18 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#19 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#20 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#21 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#22 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('create')
#23 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('blog/create')
#24 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#25 C:\xampp\htdocs\portfolio\index.php(13): CApplication->run()
#26 {main}
REQUEST_URI=/portfolio/index.php?r=blog/create
HTTP_REFERER=http://localhost:8000/portfolio/index.php?r=site/login
---
2013/07/29 01:21:56 [error] [exception.CException] exception 'CException' with message 'Property "CActiveForm.enctype" is not defined.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:173
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWidgetFactory.php(161): CComponent->__set('enctype', 'multipart/form-...')
#1 C:\xampp\htdocs\yii\framework\web\CBaseController.php(146): CWidgetFactory->createWidget(Object(BlogController), 'CActiveForm', Array)
#2 C:\xampp\htdocs\yii\framework\web\CBaseController.php(189): CBaseController->createWidget('CActiveForm', Array)
#3 C:\xampp\htdocs\portfolio\protected\views\blog\_form.php(15): CBaseController->beginWidget('CActiveForm', Array)
#4 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#5 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#6 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#7 C:\xampp\htdocs\portfolio\protected\views\blog\create.php(18): CController->renderPartial('_form', Array)
#8 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#9 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#10 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#11 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('create', Array, true)
#12 C:\xampp\htdocs\portfolio\protected\controllers\BlogController.php(79): CController->render('create', Array)
#13 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): BlogController->actionCreate()
#14 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#15 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#16 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#17 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#18 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#19 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#20 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#21 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#22 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('create')
#23 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('blog/create')
#24 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#25 C:\xampp\htdocs\portfolio\index.php(13): CApplication->run()
#26 {main}
REQUEST_URI=/portfolio/index.php?r=blog/create
HTTP_REFERER=http://localhost:8000/portfolio/index.php?r=site/login
---
2013/07/30 12:49:01 [error] [php] Undefined variable: my_model (C:\xampp\htdocs\portfolio\protected\views\blog\_form.php:57)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(869): BlogController->renderFile()
#1 C:\xampp\htdocs\portfolio\protected\views\blog\create.php(18): BlogController->renderPartial()
#2 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require()
#3 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): BlogController->renderInternal()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(869): BlogController->renderFile()
#5 C:\xampp\htdocs\yii\framework\web\CController.php(782): BlogController->renderPartial()
#6 C:\xampp\htdocs\portfolio\protected\controllers\BlogController.php(79): BlogController->render()
#7 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): BlogController->actionCreate()
#8 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#9 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): BlogController->runAction()
#10 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#12 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): BlogController->filterAccessControl()
#13 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#14 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#15 C:\xampp\htdocs\yii\framework\web\CController.php(265): BlogController->runActionWithFilters()
#16 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): BlogController->run()
#17 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#18 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#19 C:\xampp\htdocs\portfolio\index.php(13): CWebApplication->run()
REQUEST_URI=/portfolio/index.php?r=blog/create
in C:\xampp\htdocs\portfolio\protected\views\blog\_form.php (57)
in C:\xampp\htdocs\portfolio\protected\views\blog\create.php (18)
in C:\xampp\htdocs\portfolio\protected\controllers\BlogController.php (79)
2013/07/30 12:49:42 [error] [php] include(C:\xampp\htdocs\portfolio\protected\extensions\imperavi-redactor-widget\ImperaviRedactorWidget.php): failed to open stream: No such file or directory (C:\xampp\htdocs\yii\framework\YiiBase.php:395)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWidgetFactory.php(146): import()
#1 C:\xampp\htdocs\yii\framework\web\CBaseController.php(146): CWidgetFactory->createWidget()
#2 C:\xampp\htdocs\yii\framework\web\CBaseController.php(172): BlogController->createWidget()
#3 C:\xampp\htdocs\portfolio\protected\views\blog\_form.php(69): BlogController->widget()
#4 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require()
#5 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): BlogController->renderInternal()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(869): BlogController->renderFile()
#7 C:\xampp\htdocs\portfolio\protected\views\blog\create.php(18): BlogController->renderPartial()
#8 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require()
#9 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): BlogController->renderInternal()
#10 C:\xampp\htdocs\yii\framework\web\CController.php(869): BlogController->renderFile()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(782): BlogController->renderPartial()
#12 C:\xampp\htdocs\portfolio\protected\controllers\BlogController.php(79): BlogController->render()
#13 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): BlogController->actionCreate()
#14 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#15 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): BlogController->runAction()
#16 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#17 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#18 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): BlogController->filterAccessControl()
#19 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#20 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#21 C:\xampp\htdocs\yii\framework\web\CController.php(265): BlogController->runActionWithFilters()
#22 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): BlogController->run()
#23 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#24 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#25 C:\xampp\htdocs\portfolio\index.php(13): CWebApplication->run()
REQUEST_URI=/portfolio/index.php?r=blog/create
in C:\xampp\htdocs\portfolio\protected\views\blog\_form.php (69)
in C:\xampp\htdocs\portfolio\protected\views\blog\create.php (18)
in C:\xampp\htdocs\portfolio\protected\controllers\BlogController.php (79)
2013/07/30 12:50:19 [error] [exception.CException] exception 'CException' with message 'BlogController has an extra endWidget() call in its view.' in C:\xampp\htdocs\yii\framework\web\CBaseController.php:210
Stack trace:
#0 C:\xampp\htdocs\portfolio\protected\views\blog\_form.php(92): CBaseController->endWidget()
#1 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#2 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#3 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#4 C:\xampp\htdocs\portfolio\protected\views\blog\create.php(18): CController->renderPartial('_form', Array)
#5 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#6 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#7 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#8 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('create', Array, true)
#9 C:\xampp\htdocs\portfolio\protected\controllers\BlogController.php(79): CController->render('create', Array)
#10 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): BlogController->actionCreate()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#12 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#13 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#14 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#15 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#16 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#17 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#18 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#19 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('create')
#20 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('blog/create')
#21 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#22 C:\xampp\htdocs\portfolio\index.php(13): CApplication->run()
#23 {main}
REQUEST_URI=/portfolio/index.php?r=blog/create
HTTP_REFERER=http://localhost:8000/portfolio/index.php?r=site/login
---
2013/07/30 13:19:38 [error] [exception.CException] exception 'CException' with message 'Alias "ext.imperavi-redactor-widget.actions.imgUpload.RedactorUploadAction" is invalid. Make sure it points to an existing PHP file and the file is readable.' in C:\xampp\htdocs\yii\framework\YiiBase.php:316
Stack trace:
#0 C:\xampp\htdocs\yii\framework\YiiBase.php(196): YiiBase::import('ext.imperavi-re...', true)
#1 C:\xampp\htdocs\yii\framework\web\CController.php(440): YiiBase::createComponent(Array, Object(BlogController), 'imgUpload')
#2 C:\xampp\htdocs\yii\framework\web\CController.php(417): CController->createActionFromMap(Array, 'imgUpload', 'imgUpload')
#3 C:\xampp\htdocs\yii\framework\web\CController.php(259): CController->createAction('imgUpload')
#4 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('imgUpload')
#5 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('blog/imgUpload')
#6 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#7 C:\xampp\htdocs\portfolio\index.php(13): CApplication->run()
#8 {main}
REQUEST_URI=/portfolio/index.php?r=blog/imgUpload
---
2013/07/30 13:22:47 [error] [exception.CException] exception 'CException' with message 'Alias "ext.imperavi-redactor-widget.actions.imgUpload.RedactorUploadAction" is invalid. Make sure it points to an existing PHP file and the file is readable.' in C:\xampp\htdocs\yii\framework\YiiBase.php:316
Stack trace:
#0 C:\xampp\htdocs\yii\framework\YiiBase.php(196): YiiBase::import('ext.imperavi-re...', true)
#1 C:\xampp\htdocs\yii\framework\web\CController.php(440): YiiBase::createComponent(Array, Object(BlogController), 'imgUpload')
#2 C:\xampp\htdocs\yii\framework\web\CController.php(417): CController->createActionFromMap(Array, 'imgUpload', 'imgUpload')
#3 C:\xampp\htdocs\yii\framework\web\CController.php(259): CController->createAction('imgUpload')
#4 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('imgUpload')
#5 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('blog/imgUpload')
#6 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#7 C:\xampp\htdocs\portfolio\index.php(13): CApplication->run()
#8 {main}
REQUEST_URI=/portfolio/index.php?r=blog/imgUpload
---
2013/07/30 13:22:52 [error] [exception.CException] exception 'CException' with message 'Alias "ext.imperavi-redactor-widget.actions.imgUpload.RedactorUploadAction" is invalid. Make sure it points to an existing PHP file and the file is readable.' in C:\xampp\htdocs\yii\framework\YiiBase.php:316
Stack trace:
#0 C:\xampp\htdocs\yii\framework\YiiBase.php(196): YiiBase::import('ext.imperavi-re...', true)
#1 C:\xampp\htdocs\yii\framework\web\CController.php(440): YiiBase::createComponent(Array, Object(BlogController), 'imgUpload')
#2 C:\xampp\htdocs\yii\framework\web\CController.php(417): CController->createActionFromMap(Array, 'imgUpload', 'imgUpload')
#3 C:\xampp\htdocs\yii\framework\web\CController.php(259): CController->createAction('imgUpload')
#4 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('imgUpload')
#5 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('blog/imgUpload')
#6 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#7 C:\xampp\htdocs\portfolio\index.php(13): CApplication->run()
#8 {main}
REQUEST_URI=/portfolio/index.php?r=blog/imgUpload
---
2013/07/30 13:23:01 [error] [exception.CException] exception 'CException' with message 'Alias "ext.imperavi-redactor-widget.actions.imgUpload.RedactorUploadAction" is invalid. Make sure it points to an existing PHP file and the file is readable.' in C:\xampp\htdocs\yii\framework\YiiBase.php:316
Stack trace:
#0 C:\xampp\htdocs\yii\framework\YiiBase.php(196): YiiBase::import('ext.imperavi-re...', true)
#1 C:\xampp\htdocs\yii\framework\web\CController.php(440): YiiBase::createComponent(Array, Object(BlogController), 'imgUpload')
#2 C:\xampp\htdocs\yii\framework\web\CController.php(417): CController->createActionFromMap(Array, 'imgUpload', 'imgUpload')
#3 C:\xampp\htdocs\yii\framework\web\CController.php(259): CController->createAction('imgUpload')
#4 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('imgUpload')
#5 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('blog/imgUpload')
#6 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#7 C:\xampp\htdocs\portfolio\index.php(13): CApplication->run()
#8 {main}
REQUEST_URI=/portfolio/index.php?r=blog/imgUpload
---
2013/07/30 13:23:45 [error] [exception.CException] exception 'CException' with message 'Alias "ext.imperavi-redactor-widget.actions.imgUpload.RedactorUploadAction" is invalid. Make sure it points to an existing PHP file and the file is readable.' in C:\xampp\htdocs\yii\framework\YiiBase.php:316
Stack trace:
#0 C:\xampp\htdocs\yii\framework\YiiBase.php(196): YiiBase::import('ext.imperavi-re...', true)
#1 C:\xampp\htdocs\yii\framework\web\CController.php(440): YiiBase::createComponent(Array, Object(BlogController), 'imgUpload')
#2 C:\xampp\htdocs\yii\framework\web\CController.php(417): CController->createActionFromMap(Array, 'imgUpload', 'imgUpload')
#3 C:\xampp\htdocs\yii\framework\web\CController.php(259): CController->createAction('imgUpload')
#4 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('imgUpload')
#5 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('blog/imgUpload')
#6 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#7 C:\xampp\htdocs\portfolio\index.php(13): CApplication->run()
#8 {main}
REQUEST_URI=/portfolio/index.php?r=blog/imgUpload
---
2013/07/30 13:24:03 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:179
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(RedactorUploadAction), Array)
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('imgUpload')
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('blog/imgUpload')
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\xampp\htdocs\portfolio\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/portfolio/index.php?r=blog/imgUpload
---
2013/07/30 13:28:40 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:179
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(RedactorUploadAction), Array)
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('imgUpload')
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('blog/imgUpload')
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\xampp\htdocs\portfolio\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/portfolio/index.php?r=blog/imgUpload
HTTP_REFERER=http://localhost:8000/portfolio/index.php?r=blog/create
---
2013/07/30 13:29:39 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:179
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(RedactorUploadAction), Array)
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('imgUpload')
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('blog/imgUpload')
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\xampp\htdocs\portfolio\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/portfolio/index.php?r=blog/imgUpload
---
2013/07/30 13:29:49 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:179
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(RedactorUploadAction), Array)
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('imgUpload')
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('blog/imgUpload')
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\xampp\htdocs\portfolio\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/portfolio/index.php?r=blog/imgUpload
---
2013/07/30 13:29:51 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "asdf".' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('asdf')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('asdf')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('blog/asdf')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\portfolio\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/portfolio/index.php?r=blog/asdf
---
2013/07/30 13:29:53 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:179
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(RedactorUploadAction), Array)
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('imgUpload')
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('blog/imgUpload')
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\xampp\htdocs\portfolio\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/portfolio/index.php?r=blog/imgUpload
---
2013/07/30 13:29:54 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "asdf".' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('asdf')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('asdf')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('blog/asdf')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\portfolio\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/portfolio/index.php?r=blog/asdf
---
2013/07/30 13:29:55 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:179
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(RedactorUploadAction), Array)
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('imgUpload')
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('blog/imgUpload')
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\xampp\htdocs\portfolio\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/portfolio/index.php?r=blog/imgUpload
---
2013/07/30 13:29:58 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:179
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(RedactorUploadAction), Array)
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('imgUpload')
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('blog/imgUpload')
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\xampp\htdocs\portfolio\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/portfolio/index.php?r=blog/imgUpload
---
2013/07/30 13:31:25 [error] [exception.CException] exception 'CException' with message 'In order to use MIME-type validation provided by CFileValidator fileinfo PECL extension should be installed.' in C:\xampp\htdocs\yii\framework\validators\CFileValidator.php:230
Stack trace:
#0 C:\xampp\htdocs\yii\framework\validators\CFileValidator.php(172): CFileValidator->validateFile(Object(UploadedImage), 'file', Object(CUploadedFile))
#1 C:\xampp\htdocs\yii\framework\validators\CValidator.php(213): CFileValidator->validateAttribute(Object(UploadedImage), 'file')
#2 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CValidator->validate(Object(UploadedImage), NULL)
#3 C:\xampp\htdocs\portfolio\protected\extensions\imperavi-redactor-widget\actions\imgUpload\RedactorUploadAction.php(87): CModel->validate()
#4 C:\xampp\htdocs\yii\framework\web\actions\CAction.php(76): RedactorUploadAction->run()
#5 C:\xampp\htdocs\yii\framework\web\CController.php(308): CAction->runWithParams(Array)
#6 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(RedactorUploadAction))
#7 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#8 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#9 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#10 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#11 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#12 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(RedactorUploadAction), Array)
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('imgUpload')
#14 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('blog/imgUpload')
#15 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#16 C:\xampp\htdocs\portfolio\index.php(13): CApplication->run()
#17 {main}
REQUEST_URI=/portfolio/index.php?r=blog/imgUpload
HTTP_REFERER=http://localhost:8000/portfolio/index.php?r=blog/create
---
2013/07/30 13:31:35 [error] [exception.CException] exception 'CException' with message 'In order to use MIME-type validation provided by CFileValidator fileinfo PECL extension should be installed.' in C:\xampp\htdocs\yii\framework\validators\CFileValidator.php:230
Stack trace:
#0 C:\xampp\htdocs\yii\framework\validators\CFileValidator.php(172): CFileValidator->validateFile(Object(UploadedImage), 'file', Object(CUploadedFile))
#1 C:\xampp\htdocs\yii\framework\validators\CValidator.php(213): CFileValidator->validateAttribute(Object(UploadedImage), 'file')
#2 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CValidator->validate(Object(UploadedImage), NULL)
#3 C:\xampp\htdocs\portfolio\protected\extensions\imperavi-redactor-widget\actions\imgUpload\RedactorUploadAction.php(87): CModel->validate()
#4 C:\xampp\htdocs\yii\framework\web\actions\CAction.php(76): RedactorUploadAction->run()
#5 C:\xampp\htdocs\yii\framework\web\CController.php(308): CAction->runWithParams(Array)
#6 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(RedactorUploadAction))
#7 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#8 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#9 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#10 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#11 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#12 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(RedactorUploadAction), Array)
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('imgUpload')
#14 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('blog/imgUpload')
#15 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#16 C:\xampp\htdocs\portfolio\index.php(13): CApplication->run()
#17 {main}
REQUEST_URI=/portfolio/index.php?r=blog/imgUpload
HTTP_REFERER=http://localhost:8000/portfolio/index.php?r=blog/create
---
2013/07/30 13:33:24 [error] [exception.CException] exception 'CException' with message 'In order to use MIME-type validation provided by CFileValidator fileinfo PECL extension should be installed.' in C:\xampp\htdocs\yii\framework\validators\CFileValidator.php:230
Stack trace:
#0 C:\xampp\htdocs\yii\framework\validators\CFileValidator.php(172): CFileValidator->validateFile(Object(UploadedImage), 'file', Object(CUploadedFile))
#1 C:\xampp\htdocs\yii\framework\validators\CValidator.php(213): CFileValidator->validateAttribute(Object(UploadedImage), 'file')
#2 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CValidator->validate(Object(UploadedImage), NULL)
#3 C:\xampp\htdocs\portfolio\protected\extensions\imperavi-redactor-widget\actions\imgUpload\RedactorUploadAction.php(87): CModel->validate()
#4 C:\xampp\htdocs\yii\framework\web\actions\CAction.php(76): RedactorUploadAction->run()
#5 C:\xampp\htdocs\yii\framework\web\CController.php(308): CAction->runWithParams(Array)
#6 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(RedactorUploadAction))
#7 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#8 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#9 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#10 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#11 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#12 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(RedactorUploadAction), Array)
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('imgUpload')
#14 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('blog/imgUpload')
#15 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#16 C:\xampp\htdocs\portfolio\index.php(13): CApplication->run()
#17 {main}
REQUEST_URI=/portfolio/index.php?r=blog/imgUpload
HTTP_REFERER=http://localhost:8000/portfolio/index.php?r=blog/create
---
2013/07/30 13:43:39 [error] [exception.CException] exception 'CException' with message 'In order to use MIME-type validation provided by CFileValidator fileinfo PECL extension should be installed.' in C:\xampp\htdocs\yii\framework\validators\CFileValidator.php:230
Stack trace:
#0 C:\xampp\htdocs\yii\framework\validators\CFileValidator.php(172): CFileValidator->validateFile(Object(UploadedImage), 'file', Object(CUploadedFile))
#1 C:\xampp\htdocs\yii\framework\validators\CValidator.php(213): CFileValidator->validateAttribute(Object(UploadedImage), 'file')
#2 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CValidator->validate(Object(UploadedImage), NULL)
#3 C:\xampp\htdocs\portfolio\protected\extensions\imperavi-redactor-widget\actions\imgUpload\RedactorUploadAction.php(87): CModel->validate()
#4 C:\xampp\htdocs\yii\framework\web\actions\CAction.php(76): RedactorUploadAction->run()
#5 C:\xampp\htdocs\yii\framework\web\CController.php(308): CAction->runWithParams(Array)
#6 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(RedactorUploadAction))
#7 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#8 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#9 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#10 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#11 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#12 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(RedactorUploadAction), Array)
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('imgUpload')
#14 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('blog/imgUpload')
#15 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#16 C:\xampp\htdocs\portfolio\index.php(13): CApplication->run()
#17 {main}
REQUEST_URI=/portfolio/index.php?r=blog/imgUpload
HTTP_REFERER=http://localhost:8000/portfolio/index.php?r=blog/create
---
2013/07/30 14:02:36 [error] [php] include(User.php): failed to open stream: No such file or directory (C:\xampp\htdocs\yii\framework\YiiBase.php:421)
Stack trace:
#0 C:\xampp\htdocs\portfolio\protected\components\UserIdentity.php(15): spl_autoload_call()
#1 C:\xampp\htdocs\portfolio\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#2 C:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate()
#3 C:\xampp\htdocs\yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute()
#4 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CInlineValidator->validate()
#5 C:\xampp\htdocs\portfolio\protected\controllers\SiteController.php(94): LoginForm->validate()
#6 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#8 C:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#9 C:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#10 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#11 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#12 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#13 C:\xampp\htdocs\portfolio\index.php(13): CWebApplication->run()
REQUEST_URI=/portfolio/index.php?r=site/login
in C:\xampp\htdocs\portfolio\protected\components\UserIdentity.php (15)
in C:\xampp\htdocs\portfolio\protected\models\LoginForm.php (52)
in C:\xampp\htdocs\portfolio\protected\controllers\SiteController.php (94)
2013/07/30 14:12:40 [error] [php] include(User.php): failed to open stream: No such file or directory (C:\xampp\htdocs\yii\framework\YiiBase.php:421)
Stack trace:
#0 C:\xampp\htdocs\portfolio\protected\components\UserIdentity.php(15): spl_autoload_call()
#1 C:\xampp\htdocs\portfolio\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#2 C:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate()
#3 C:\xampp\htdocs\yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute()
#4 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CInlineValidator->validate()
#5 C:\xampp\htdocs\portfolio\protected\controllers\SiteController.php(94): LoginForm->validate()
#6 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#8 C:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#9 C:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#10 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#11 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#12 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#13 C:\xampp\htdocs\portfolio\index.php(13): CWebApplication->run()
REQUEST_URI=/portfolio/index.php?r=site/login
in C:\xampp\htdocs\portfolio\protected\components\UserIdentity.php (15)
in C:\xampp\htdocs\portfolio\protected\models\LoginForm.php (52)
in C:\xampp\htdocs\portfolio\protected\controllers\SiteController.php (94)
2013/07/30 16:37:26 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[23000]: Integrity constraint violation: 1452 Cannot add or update a child row: a foreign key constraint fails (`decorvinilo`.`entradas`, CONSTRAINT `entradas_ibfk_1` FOREIGN KEY (`autor`) REFERENCES `usuarios` (`id`)). The SQL statement executed was: INSERT INTO `entradas` (`titulo`, `texto`, `fecha_publicacion`) VALUES (:yp0, :yp1, NOW()).
in C:\xampp\htdocs\portfolio\protected\controllers\BlogController.php (73)
in C:\xampp\htdocs\portfolio\index.php (13)
2013/07/30 16:37:26 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[23000]: Integrity constraint violation: 1452 Cannot add or update a child row: a foreign key constraint fails (`decorvinilo`.`entradas`, CONSTRAINT `entradas_ibfk_1` FOREIGN KEY (`autor`) REFERENCES `usuarios` (`id`)). The SQL statement executed was: INSERT INTO `entradas` (`titulo`, `texto`, `fecha_publicacion`) VALUES (:yp0, :yp1, NOW())' in C:\xampp\htdocs\yii\framework\db\CDbCommand.php:357
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(1023): CDbCommand->execute()
#1 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(795): CActiveRecord->insert(NULL)
#2 C:\xampp\htdocs\portfolio\protected\controllers\BlogController.php(73): CActiveRecord->save()
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): BlogController->actionCreate()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('create')
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('blog/create')
#14 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#15 C:\xampp\htdocs\portfolio\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/portfolio/index.php?r=blog/create
HTTP_REFERER=http://localhost:8000/portfolio/index.php?r=blog/create
---
